Maddori.Apple/Maddori.Apple/Screen/MyFeedback/MyFeedback/MyFeedbackViewController.swift

```diff
@@ -113,6 +145,27 @@ final class MyFeedbackViewController: BaseViewController {
         }
     }
     
+    private func setLayoutEmptyView() {
+        view.addSubview(emptyView)
+        emptyView.snp.makeConstraints {
+            $0.top.equalTo(myFeedbackLabel.snp.bottom)
+            $0.leading.trailing.bottom.equalTo(view.safeAreaLayoutGuide)
```

Comment:
p5;
leading, trailing은 horizontalEdges,
top, bottom은 verticalEdges 로 한 번에 나타낼 수 있답니다!

고로
```suggestion
            $0.horizontalEdges.bottom.equalTo(view.safeAreaLayoutGuide)
```
로 합칠 수 있다는거죠 하핳

Reply:
사실 .. 저도 이거에 대해서 고민이 많은데요..
horizontalEdges는 left.right을 한 번에 쓰는 거라 명백히 말하면 leading.trailing 와 다르지만
제가 알고 있는 left(right) 와 leading(trailing)의 차이점은 글 읽는 방향? 정도 뿐이라 어떤 것을 사용하는 게 올바른지 모르겠어요!

이거에 대해 여러분들과 이야기 나눠보고 싶네요!!!! 😁

Reply:
horizontalEdges 이 left, right 에 대한건지 몰랐네요..ㅎㅎ 

저희는 지금까지 사실 leading, trailing 으로 레이아웃을 잡아왔으니 horizontalEdges이 left, right 이라는 것을 알게됐으니 안쓰는게 좋다고 생각합니다! (저는)

Reply:
와 진짜 대박 저는 사실 horizontalEdges가 left, right을 말하는건지 몰랐어요... 생각 없이 그냥 '음 양 옆 잡아주는걸 하나로 통합 가능하구나~' 하고 쓰고 있었는데 left right 였군요...

그걸 확인하러 cmd + control 로 snapkit 세계에 모험을 떠났는데 웬걸...
leading, trailing 합쳐주는 것도 있네요..!!!!!!

<img width="550" alt="image" src="https://user-images.githubusercontent.com/72431640/211134270-f77bb105-409d-4b75-ae59-6898003e1444.png">

`directionalHorizontalEdges`...
근데 사실 코드 글자 하나라도 줄이려고 하나로 묶자는 의미도 있었는데 이렇게 되면 더 길어지는군요 하핳

오늘도 새로운걸 배워갑니닷...

Reply:
오호 `directionalHorizontalEdges`라는 것도 있군요 ㅋㅋㅋ 공유해줘서 감사합니다! 싱기방기

Reply:
막연히 좌우라고 생각은 했는데, `leading` `trailing`이 아닌 `left` `right` 일거라곤 생각 못했네요 !
어차피 양쪽 다 잡는거라면 `horizontalEdges`써도 상관없을 것 같긴한데, 현재는 `leading` `trailing`으로 계속 사용했었으니 이대로 가는게 좋아보입니다 !
추후에 다른 프로젝트를 하게 된다거나, 전체적으로 수정할 경우가 생기면 적용해보면 좋을 것 같습니다 !
정보 감사합니다 이드 ^________^